change/@microsoft-teams-js-d6954dd2-027c-4a76-b6b4-7f9eb86a815a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Move from Yarn to Pnpm",
```

Comment:
Please use past tense, e.g., "Migrated from Yarn to pnpm for repository package management."